docs/05-engineering/security-incidents.md

```diff
@@ -0,0 +1,49 @@
+# Security incidents
+
+Something went "bump" in the night (or the day)? This document explains what to do when responding to a security incident. See [What is an incident?](#what-is-an-incident) if you need help determining whether something counts as an incident.
+
+## Reporting phishing emails
+If you receive a phishing email, follow these steps to report to CivicActions IT:
+
+1. Do not click any links in the email. Do not delete it yet. You may mark it as spam.
+1. If you can, click the `Show Original` option in the "triangle" menu associated with the email. This will open a new window with the Original Message above and the raw text of the email below.
+1. Click on the `Download Original` link halfway down the page and it will save text of the email to your hard drive.
+1. Mark the email as a phishing email by selecting the `Report phishing` option in the same "triangle" menu associated with the email.
+1. Forward the email to [support@civicactions.com](mailto:support@civicactions.com). As long as you haven't clicked on link or downloaded the file, you may stop here.
+   * If you suspect that the email has compromised your system with a download or link, attach the original text you downloaded. Please include *Security Incident* in the subject line, along with a brief description of the issue (Ex. Clicked on link in phishing email).
+1. Report the phishing email in the [#general](https://civicactions.slack.com/messages/general/) Slack channel (if you have access).
```

Comment:
Wondering if this should go to "Loving Security" and then an announcement of said phishing